sdk/lib/rtl/threadpool.c

```diff
@@ -23,16 +23,12 @@
 #ifdef __REACTOS__
 #include <rtl_vista.h>
 #define NDEBUG
-#include "wine/list.h"
-#include <debug.h>
-
-#define ERR(fmt, ...)    DPRINT1(fmt, ##__VA_ARGS__)
-#define FIXME(fmt, ...)  DPRINT(fmt, ##__VA_ARGS__)
-#define WARN(fmt, ...)   DPRINT(fmt, ##__VA_ARGS__)
-#define TRACE(fmt, ...)  DPRINT(fmt, ##__VA_ARGS__)
-#ifndef ARRAY_SIZE
-#define ARRAY_SIZE(_x) (sizeof((_x))/sizeof((_x)[0]))
-#endif
+#include <winnt.h>
+#include <assert.h>
```

Comment:
Do we really need these two here, instead of having a very minimal list of definitions?